deploy/olm-catalog/knative-serving-operator/0.6.0/serving_v1alpha1_knativeserving_crd.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: installs.serving.knative.dev
+  name: knativeservings.serving.knative.dev
 spec:
   group: serving.knative.dev
   names:
-    kind: Install
-    listKind: InstallList
-    plural: installs
-    singular: install
+    kind: KnativeServing
+    listKind: KnativeServingList
+    plural: knativeservings
+    singular: knativeserving
     shortNames:
     - ksi
```

Comment:
@jcrossley3 should that `ksi` be `ks` only  ?

Reply:
Good call!

Reply:
OK, I will update my PR w/ `ke` .. fwiw